services/scorecard/scorecard.tester.js

```diff
@@ -0,0 +1,10 @@
+import { createServiceTester } from '../tester.js'
+export const t = await createServiceTester()
+
+t.create('score (valid)')
+  .get('/github.com/rohankh532/org-workflow-add.json')
+  .expectBadge({
+    label: 'scorecard score',
+    message: '4.2',
+    color: 'orange',
+  })
```

Comment:
It's good to have a live test (see also our [guide on writing service tests](https://github.com/badges/shields/blob/master/doc/service-tests.md)), but we don't typically want to pin a live test to a specific value because that can make the test brittle. Instead, we follow a similar pattern as we do for validating response schemas and define the _type_ of result we'd expect to see, typically with another schema.

In this case I think `message: Joi.number().min(0)` would suffice instead of the explicit `4.2` value

Reply:
Fixed this, along with assuming the color value

Reply:
Fixed this, along with assuming the color value